assets/constants.ts

```diff
@@ -651,7 +651,7 @@ export = {
   "ENABLE_ACCOUNT_DELETION": false,
 
   // Used to hide the callout to classroom until it is ready.
-  "SHOW_CLASSROOM_CALLOUT": false,
+  "SHOW_CLASSROOM_CALLOUT": true,
```

Comment:
Please leave this as false since we want to hide the classroom changes behind this flag until everything is ready.

Reply:
Sure, will update it,thank you for mentioning.